industrial_trajectory_filters/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8.3)
 
 project(industrial_trajectory_filters)
 
+add_compile_options(-std=c++11)
```

Comment:
`add_compile_options` is to `target_compile_options` as `include_directories` is to `target_include_directories`.
If I understand it correctly, mixing options between targets might lead to ABI incompabilities, so the per-directory/package directive should be less error-prone.